### PR TITLE
Added OutboundNAT policy support for Windows

### PR DESCRIPTION
**Pull request by nwoodmsft**
This change adds support for OutboundNAT policy to the Windows driver passed via a generic option.

### CODE REVIEW

**Comment by pradipd** [comment]
LGTM

**Comment by pradipd** [comment]
ping @fcrisciani @mavenugo ; Flavio, Madhu, can you review please?  Note that this change affects windows only.

File: drivers/windows/windows.go [hunk 1]
```diff
@@ -208,6 +211,18 @@ func (d *driver) parseNetworkOptions(id string, genericOptions map[string]string
 				return nil, err
 			}
 			config.VSID = uint(vsid)
+		case EnableOutboundNat:
+			if system.GetOSVersion().Build <= 16236 {
```
Comment (ddebroy) [inline drivers/windows/windows.go]: Can the build number `16236` be defined as a global constant? It seems to be used at a couple of places.

**Comment by fcrisciani** [comment]
Comment can be addressed in a separate Pr, merging this one

**Pull request merged**